tests/src/core/getExports.js

```diff
@@ -11,6 +11,12 @@ import * as fs from 'fs';
 import { getFilename } from '../utils';
 import { test as testUnambiguous } from 'eslint-module-utils/unambiguous';
 
+const doesBabelHaveVisitorKeys = () => {
+  const babelPath = require.resolve('babel-eslint');
```

Comment:
is there a reason not to do this at module level, and also use it on line 30?

Reply:
Not a strong reason.  I generally default to keeping control variables like that at the lowest level needed and not adding a bunch to the module scope.  But I don't have a strong opinion in this case, and can change it if you prefer.

Reply:
It should have a slight performance benefit, and it eliminates the chance that the resolution could be different, so i think it'd be good to make the change.

Reply:
Updated